### DISCUSSION TITLE
Feature Request: Additional labels in AutoConnectLabels.h for translations

### DISCUSSION THREAD

**Discussion by faanskit** [Hieromon/AutoConnect#563] [Feature Request]
Kindly ask to include three new labels,
I.e.,

```c++
// Page AutoConnectConfigAux text: Prev.
#ifndef AUTOCONNECT_PAGECONFIG_PREVIOUS
#define AUTOCONNECT_PAGECONFIG_PREVIOUS "Prev."
#endif // !AUTOCONNECT_PAGECONFIG_PREVIOUS

// Page AutoConnectConfigAux text: Next
#ifndef AUTOCONNECT_PAGECONFIG_NEXT
#define AUTOCONNECT_PAGECONFIG_NEXT "Next"
#endif // !AUTOCONNECT_PAGECONFIG_NEXT

// Page AutoConnectConfigAux text: Next
#ifndef AUTOCONNECT_TEXTLABEL_RESET
#define AUTOCONNECT_TEXTLABEL_RESET " in progress..."
#endif // !AUTOCONNECT_TEXTLABEL_RESET

```
And have them used in the following places:

**AutoConnectPageImpl.hpp line 1232/1237**
```c++
  // Prepare perv. button
  if (page >= 1) {
    snprintf_P(slBuf, bufSize - (slBuf - ssidList), (PGM_P)_ssidPage, page - 1, PSTR("Prev."));
    slBuf = ssidList + strlen(ssidList);
  }
  // Prepare next button
  if (validCount > (page + 1) * AUTOCONNECT_SSIDPAGEUNIT_LINES) {
    snprintf_P(slBuf, bufSize - (slBuf - ssidList), (PGM_P)_ssidPage, page + 1, PSTR("Next"));
  }
```
==>
```c++
  // Prepare perv. button
  if (page >= 1) {
    snprintf_P(slBuf, bufSize - (slBuf - ssidList), (PGM_P)_ssidPage, page - 1, PSTR(AUTOCONNECT_PAGECONFIG_PREVIOUS));
    slBuf = ssidList + strlen(ssidList);
  }
  // Prepare next button
  if (validCount > (page + 1) * AUTOCONNECT_SSIDPAGEUNIT_LINES) {
    snprintf_P(slBuf, bufSize - (slBuf - ssidList), (PGM_P)_ssidPage, page + 1, PSTR(AUTOCONNECT_PAGECONFIG_NEXT));
  }
```
**AutoConnectCoreImpl.hpp - Line 1208**
```c++
template<typename T>
String AutoConnectCore<T>::_induceReset(PageArgument& args) {
  AC_UNUSED(args);
  _rfReset = true;
  return String(F(AUTOCONNECT_BUTTONLABEL_RESET " in progress..."));
}
```
==>
```c++
template<typename T>
String AutoConnectCore<T>::_induceReset(PageArgument& args) {
  AC_UNUSED(args);
  _rfReset = true;
  return String(F(AUTOCONNECT_BUTTONLABEL_RESET AUTOCONNECT_TEXTLABEL_RESET));
}
```

**Comment by Hieromon** [comment]
Please submit a PR with your change intentions.

P.S.
Please post Feature requests to Discussions, as closing an Issue makes it difficult for other users to trace the history of what requests were made in the past. Discussions have been categorized for that.
Thank you.

**Reply by faanskit** [reply]
I don't have access to my system until next week, but have created a branch in my fork and are ready to create a PR.
But, since I have not tested it I will wait until (1) you agree that I can create the PR on these premises or (2) I have tested it.

The changes can be found [here](https://github.com/Hieromon/AutoConnect/compare/master...faanskit:AutoConnect:lang_add
).

Let me know if you want me to push it as-is, or wait util I've tested it.

**Reply by Hieromon** [reply]
Ok, please test when you have time. You can a PR to the master. I will review a tested PR, then run the usual tests on my local CI. If there are no problems, I will merge it into the patches/553 branch.

I should be able to release it as v1.4.2 after the documentation is in place within the next week.

Thank you for your contribution.

**Reply by faanskit** [reply]
PR #564 created

**Comment by Hieromon** [comment]
Merged #575
